db-backup/create-db-dump.sh

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-set +x  # Don't log credentials
-DB_URI=$(echo $VCAP_SERVICES | jq -r '.postgres[0].credentials.uri')
-set -x  # Restore logging
+DB_URI=$(echo $VCAP_SERVICES | jq -r '.postgres[0].credentials.uri') >/dev/null # Don't print credentials to stdout
```

Comment:
I'm not sure I understand why the `> /dev/null` is needed; if I do `TEST=$(echo hello world)` on my machine nothing gets echoed to stdout, so why is stuff getting echoed here? 🤔

Reply:
I think it shows up as part of `stderr` in the error case i.e. here:  https://ci.marketplace.team/job/database-backup/1137/console

Reply:
the `ERR` prefix maps to `stderr` as per https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html

Reply:
So should we be redirecting `stderr` instead of `stdout`?

Reply:
Possibly, that said I'm not sure if that'd work either thinking more about it. If it is just dumping the command history.